usuarios/delete-usuarios.spec.js

```diff
@@ -0,0 +1,31 @@
+const user = require('../utils/userData')
+const testServer = require('../utils/testServer')
+const rota = require('../utils/rotas')
+
+var usuario
+var user_id
```

Comment:
```suggestion
var usuario
var userId
```
no JS o padrão é camel case, por isso não usamos _ pra nomear